proxy/src/main/java/org/apache/accumulo/proxy/ProxyServer.java

```diff
@@ -189,31 +188,23 @@ public void onRemoval(RemovalNotification<UUID,ConditionalWriter> notification)
 
   private final ThriftServerType serverType;
 
-  public ProxyServer(Properties props) {
+  public ProxyServer(Properties proxyProps, Properties clientProps) {
 
-    String useMock = props.getProperty("useMockInstance");
+    String useMock = proxyProps.getProperty("useMockInstance");
     if (useMock != null && Boolean.parseBoolean(useMock))
       instance = DeprecationUtil.makeMockInstance(this.getClass().getName());
     else {
-      ClientConfiguration clientConf;
-      if (props.containsKey("clientConfigurationFile")) {
-        String clientConfFile = props.getProperty("clientConfigurationFile");
-        clientConf = ClientConfiguration.fromFile(new File(clientConfFile));
-      } else {
-        clientConf = ClientConfiguration.loadDefault();
-      }
-      instance = new ZooKeeperInstance(clientConf.withInstance(props.getProperty("instance"))
-          .withZkHosts(props.getProperty("zookeepers")));
+      instance = new ZooKeeperInstance(ClientConfConverter.toClientConf(clientProps));
```

Comment:
Do we need to convert the properties to the old type? Can't we just use the new connector factory?

Reply:
We still need to create an `Instance` here until #452 and #414 are complete.

Reply:
I had figured we could call `conn.getInstance()`, but now I see there isn't a Connector at this point in the code, and I'm now realizing it's because the Proxy allows (re-)login on each proxied API call. I think that's probably a bit overkill. I think the Proxy could be simplified by operating as a single-user client, but that's outside the scope of this issue. Perhaps that could be done as part of a move to separate the proxy from Accumulo's core packaging.